### PR TITLE
fix: browser and client auth issues

### LINKED ISSUE #837: Configure focus in register/login menu
1) Autofocus when register menu arrives
    **P.S. I know that you already tried this and it didn't work since the game closes CursorMenu unexpectedly.**
    **Please find a workaround for this. For example, if cursor menu is not open, setFocused(false) and setFocused(true)**
    ** I hope it helps you with repairing cursor**

2) In the race menu, unfocus
3) When logined or registered, unfocus

### LINKED ISSUE #874: Fix race between page load and client register load

### LINKED ISSUE #896: Error: Form with id 0 doesn't exist
Sometimes after logging in with a new account, you're not spawned in the world. Logging in with the same account again fixes it, you spawn in the right place. You also can't see the character selection menu. After restarting server and rejoining it, players see character selection menu being in the location they left.

```
[2022-01-30 01:29:25.552] [console] [info] getUserProfileId { user: { id: 62 } }
[2022-01-30 01:29:25.555] [console] [error] Error: Form with id 0 doesn't exist
    at t.createActor (/skymp5-server/ts/nativeGameServer.ts:21:33)
    at EventEmitter.<anonymous> (/skymp5-server/ts/systems/spawn.ts:24:27)
    at EventEmitter.emit (node:events:390:28)
    at /skymp5-server/ts/systems/login.ts:58:18
    at processTicksAndRejections (node:internal/process/task_queues:96:5) Promise {
  <rejected> Error: Form with id 0 doesn't exist
      at t.createActor (/skymp5-server/ts/nativeGameServer.ts:21:33)
      at EventEmitter.<anonymous> (/skymp5-server/ts/systems/spawn.ts:24:27)
      at EventEmitter.emit (node:events:390:28)
      at /skymp5-server/ts/systems/login.ts:58:18
      at processTicksAndRejections (node:internal/process/task_queues:96:5)
}
```

### PR DESCRIPTION
**Pull request by kkEngine**
closes #837, closes #874, closes #896

### CODE REVIEW

File: skymp5-front/src/index.js [hunk 1]
```diff
@@ -33,3 +33,5 @@ window.scrollToLastMessage = () => {
   const _list = document.querySelector('#chat > .list');
   if (_list != null) { _list.scrollTop = _list.offsetHeight * _list.offsetHeight; }
 };
+
+window.skyrimPlatform.sendMessage("front-loaded");
```
Comment (Pospelove) [inline skymp5-front/src/index.js:37]: if (window.skyrimPlaatform !== undefined)

**Review by Pospelove — DISMISSED** [review]
otherwise LGTM if it works

**Comment by nic11** [comment]
PR title should probably be:
`fix: browser and client auth issues`
since it also affects skymp5-front

File: skymp5-client/src/authSystem.ts [hunk 2]
```diff
@@ -53,7 +53,10 @@ export const main = (lobbyLocation: Transform): void => {
   if (isOfflineMode) {
     onAuthListeners({ local: { profileId: settingsGameData.profileId } });
   } else {
-    loadLobby(lobbyLocation);
+    startListenBrowserMessage();
+    browser.addOnWindowLoadListener(() => {
+      if (isListenBrowserMessage) loadLobby(lobbyLocation)
```
Comment (nic11) [inline skymp5-client/src/authSystem.ts:58]: missing semicolon?

**Comment by nic11** [comment]
This probably also fixes #896?

File: skymp5-front/src/index.js (hunk 1 above)
Comment (kkEngine) [reply skymp5-front/src/index.js:37]: We have this https://github.com/skyrim-multiplayer/skymp/blob/8c589db687030db883cdcc8fb34d78e62a4bb1f9/skymp5-front/src/index.js#L15

**Review by nic11 — APPROVED** [review]

File: skymp5-front/src/index.js (hunk 1 above)
Comment (nic11) [reply skymp5-front/src/index.js]: Added `if (window.skyrimPlatform?.sendMessage)`

**Pull request merged**